### ISSUE TITLE
Should the RegExp.$1 getter check the value of its this value?

### ISSUE THREAD

**Issue by claudepache** [claudepache/es-regexp-legacy-static-properties#7]
That is:

``` js
MyRegExp extends RegExp {  }
MyRegExp.$1 // should it give by default the same result of RegExp.$1?
```

**Comment by claudepache** [comment]
Just resolved in 5e0f68fba964e689ec71897819232e93a3620f7d. Since the feature is disabled for the subclass, it is logical to make `MyRegExp.$1` appear to not exist (i.e., it evaluates to undefined).

**Comment by claudepache** [comment]
... but leaving this issue open, as it is currently not implemented by any browser.
